packages/wonder-blocks-form/src/components/text-field.md

```diff
@@ -54,31 +54,215 @@ class TextFieldExample extends React.Component {
 <TextFieldExample />
 ```
 
-Password
+Password (with Validation)
 
 ```js
+import {StyleSheet} from "aphrodite";
+import {View, Text} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {TextField} from "@khanacademy/wonder-blocks-form";
 
 class TextFieldExample extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
             value: "Password123",
+            errorMessage: null,
         };
+        this.handleOnChange = this.handleOnChange.bind(this);
+        this.handleValidation = this.handleValidation.bind(this);
+        this.handleOnError = this.handleOnError.bind(this);
```

Comment:
Ugh this is unfortunate, but hopefully we'll avoid using this once we fully migrate to Storybook 🤞

Reply:
🤞